test/commands/multi.spec.js

```diff
@@ -64,9 +64,9 @@ describe("The 'multi' method", function () {
                     // Provoke an error at queue time
                     multi1 = client.MULTI();
                     multi1.mset("multifoo", "10", "multibar", "20", helper.isString("OK"));
-                    multi1.set("foo2", helper.isError());
-                    multi1.incr("multifoo", helper.isNumber(11));
-                    multi1.incr("multibar", helper.isNumber(21));
+                    multi1.set("foo2");
+                    multi1.incr("multifoo");
+                    multi1.incr("multibar");
```

Comment:
I did some further testing, `multi1.incr("multibar")` doesn't ever invoke a callback -- this test was a noop.

`multi1.set("foo2")` does result in an error being returned on both old and new redis', so I'm adding this back.